charts/govuk-apps-conf/templates/static-nginx-configmap.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: static-nginx-conf
+  labels:
+    {{- include "govuk-apps-conf.labels" . | nindent 4 }}
+    app.kubernetes.io/component: web
+data:
+  nginx.conf: |-
+    user nginx;
+
+    error_log  /var/log/nginx/error.log warn;
+    pid        /var/run/nginx.pid;
+
+    events {
+      worker_connections  1024;
+    }
+
+    http {
+      include       /etc/nginx/mime.types;
+      default_type  application/octet-stream;
+
+      server_tokens off;
+
+      sendfile        on;
+      keepalive_timeout  65;
+
+      upstream {{ .Release.Name }} {
+        server 127.0.0.1:{{ .Values.appPort }};
+      }
+
+      server {
+        listen {{ .Values.nginxPort }};
+
+        location / {
+          proxy_set_header   Host $http_host;
+          proxy_set_header   X-Real-IP $remote_addr;
+          proxy_pass         http://{{ .Release.Name }};
+          proxy_redirect     off;
+        }
+        location /assets {
+          proxy_set_header   Authorization "";
+          proxy_set_header   Connection "";
+          proxy_set_header   X-Real-IP $remote_addr;  # TODO: pass the actual end-client address
+          proxy_hide_header  x-amz-id-2;
+          proxy_hide_header  x-amz-meta-server-side-encryption;
+          proxy_hide_header  x-amz-request-id;
+          proxy_hide_header  x-amz-server-side-encryption;
+          proxy_hide_header  x-amz-version-id;
+          add_header         Cache-Control max-age=31536000;
+          proxy_intercept_errors on;
+          proxy_pass         https://govuk-app-assets-{{ .Values.govukEnvironment }}.s3.eu-west-1.amazonaws.com;
+        }
+      }
+
+      # Google site verification
+      location /googlec908b3bc32386239.html {
+        return 200 'google-site-verification: googlec908b3bc32386239.html';
+      }
+
+      # Endpoint that isn't cached, which is used to assert that an external
+      # service can receive a response from GOV.UK origin on assets hostname. It
+      # is intended for pingdom monitoring
+      location = /__canary__ {
```

Comment:
I don't think we need `/__canary__` here. It only needs to be on Router and assets-origin.

Reply:
I added it because the current assets-origin nginx config also has it. I can get rid of it if it isn't required